tremor-script/src/registry.rs

```diff
@@ -118,7 +118,9 @@ pub fn registry() -> Registry {
         }))
         .insert(tremor_fn!(system|version(_context) {
             Ok(Value::String(VERSION.into()).into_static())
-        }));
+        }))
+        .insert(tremor_fn!(logging|info(_context) {
```

Comment:
I think this function is already being registered in `tremor-pipeline/src/logging.rs`, so no need to register it twice.

Reply:
this seems to be fixed can we resolve this?